pages/management/applications.md

```diff
@@ -66,6 +66,40 @@ When you create a view, you are saving a specific set of filters that you may wa
 
 You will be asked to choose a name for the saved view. This view can then be selected at any time by clicking the *Views* button above the filter box.
 
+## Device tags
+
+Tags provide an easy way to include more information about your devices than what is already provided in the dashboard. A device can be given multiple tags, and the tags can be used when you create filters.
```

Comment:
it may be worth adding an example of this to the section of the docs on filters, since the combination of tags + filters is so powerful